djstripe/stripe_objects.py

```diff
@@ -406,27 +411,29 @@ class StripeSource(PolymorphicModel, StripeObject):
 
 class StripeCharge(StripeObject):
     """
-To charge a credit or a debit card, you create a charge object. You can
-retrieve and refund individual charges as well as list all charges. Charges
-are identified by a unique random ID. (Source: https://stripe.com/docs/api/python#charges)
-
-# = Mapping the values of this field isn't currently on our roadmap.
-    Please use the stripe dashboard to check the value of this field instead.
-
-Fields not implemented:
-
-* **object** - Unnecessary. Just check the model name.
-* **application_fee** - #. Coming soon with stripe connect functionality
-* **balance_transaction** - #
-* **dispute** - #; Mapped to a ``disputed`` boolean.
-* **fraud_details** - Mapped to a ``fraudulent`` boolean.
-* **order** - #
-* **receipt_email** - Unnecessary. Use Customer.email. Create a feature request if this is functionality you need.
-* **receipt_number** - Unnecessary. Use the dashboard. Create a feature request if this is functionality you need.
-* **refunds** - #
-* **source_transfer** - #
-
-.. attention:: Stripe API_VERSION: model fields and methods audited to 2016-03-07 - @kavdev
+    A representation of a Stripe charge object.
```

Comment:
The lack of indentation is actually necessary to ensure proper doc rendering. Please revert this for other models as well.